src/ensembl/src/shared/components/panel/Panel.scss

```diff
@@ -14,11 +14,14 @@
   border-top-left-radius: 5px;
   border-top-right-radius: 5px;
   background-color: $light-grey;
-  box-shadow: 0px 1px 3px $medium-light-grey;
+  box-shadow: 0 4px 3px -2px $medium-light-grey;
   height: 34px;
   padding: 4px 10px;
   width: 100%;
   line-height: 24px;
+  position: sticky;
```

Comment:
What effect will this rule have?

Reply:
We have box shadow on the subtab panel which on scroll is visible as in the picture. Now we have shadow to the bottom.
![Screenshot 2020-09-04 at 08 48 11](https://user-images.githubusercontent.com/6347854/92215003-15fdad00-ee8c-11ea-8d86-c32d6cc886e6.png)
![Screenshot 2020-09-04 at 08 48 06](https://user-images.githubusercontent.com/6347854/92215005-16964380-ee8c-11ea-9203-24f2d9ea3bdb.png)

Reply:
Ok, I see now what the position: sticky is doing for the panel. It looks cool, but there are two things to consider:

- First, `Panel` is a shared component. I am almost sure its header should not always have `position: sticky`. I think, if you really want to use the `position: sticky`, you should customise the header through Panel's [`classNames` props](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/components/panel/Panel.tsx#L29)
- The scrolling solution with `posititon: sticky` is very neat. It wouldn't have occurred to me. I would have probably fixed the height of the panel and used `overflow-y: auto` on it. The result would be less pretty (there would be an extra scrollbar inside the panel), but possibly more robust. I wonder whether the solution with `position:sticky` will present problems when we will try to develop the code for scrolling to the selected protein when user clicks on the protein id on a transcript.